retriever/__main__.py

```diff
@@ -109,21 +109,36 @@ def main():
             # scripts should never be empty because check_for_updates is run on SCRIPT_LIST init
             if not (args.l or args.k or isinstance(args.v, list)):
                 all_scripts = dataset_names()
-                print("Available datasets : {}\n".format(len(all_scripts)))
                 from retriever import lscolumns
-
-                lscolumns.printls(all_scripts)
+                all_scripts_combined = []
+                for dataset in all_scripts['offline']:
+                    all_scripts_combined.append((dataset, True))
+                for dataset in all_scripts['online']:
+                    if dataset in all_scripts['offline']:
+                        continue
+                    all_scripts_combined.append((dataset, False))
+                all_scripts_combined = sorted(all_scripts_combined, key = lambda x: x[0])
+                print("Available datasets : {}\n".format(len(all_scripts_combined)))
+                lscolumns.printls(all_scripts_combined)
+                print("\nred: online datasets")
+                print("To see the full list of available datasets, visit "
```

Comment:
Do we need the red?

Reply:
The word `red`?

Reply:
@henrykironde This was added in to show which scripts were "offline" when you retriever ls. I think it will make a bit more sense when @harshitbansal05 adds in some other keys and comments for the ls. Basically we want the user to be able to know which scripts they have locally and which ones they do not.